core/domain/change_domain.py

```diff
@@ -164,6 +181,11 @@ def validate_dict(self, change_dict):
                 valid_cmd_attribute_specs = copy.deepcopy(cmd)
                 break
 
+        for cmd in self.DEPRECATED_COMMANDS:
+            if cmd == cmd_name:
+                raise utils.DeprecatedCommandError(
+                    'Command %s is deprecated' % cmd_name)
```

Comment:
```suggestion
        if cmd_name in self.DEPRECATED_COMMANDS:
                raise utils.DeprecatedCommandError(
                    'Command %s is deprecated' % cmd_name)
```

Can we do this (for better readability)?

Reply:
Done.